### DISCUSSION TITLE
I can help build Mac Arm binaries

### DISCUSSION THREAD

**Discussion by anonymousaga** [supermerill/SuperSlicer#3986] [General]
Supermill, if you'd like, I can help build arm binaries for Macos. I have a decent understanding of terminal/Unix/compiling and a basic understanding of C.

I also have an m1-based mac.

**Comment by supermerill** [comment]
Hi
Currently, my biggest problem with the mac build is to retreive my apple dev keys, and refine the process to sign the app.
Someone already submitted a fixed workflow, i'll have to test it, but I'll let it slip a bit more.
If you want to help me
. fork the project
. add https://github.com/supermerill/SuperSlicer/pull/3755 to your fork
. add the necessary secret variables so your apple ids are available for the workflow
. build mac (intel) (launch the rc mac intel action)
. test it, it should be signed and all.

when mac intel is build, the enxt step is to build the arm version. github just has made available mac arm runner, so by changing just the runner, it may works as-is.

**Comment by anonymousaga** [comment]
I don't have apple developer account, so I think I can't help.
